lexos/templates/kmeans.html

```diff
@@ -2,443 +2,147 @@
 {% set active_page = 'kmeans' %}
 
 {% block head %}
-<script type="text/javascript" src="{{ url_for('static', filename='js/scripts_kmeans.js') }}?ver={{version}}"></script>
-<script type="text/javascript">
-	var dataset 	    = {{ kmeansIndex }};
-	var fileNameStr     = "{{ fileNameStr }}";
-	var tablelabels     = fileNameStr.split("#");
-	var totalFileNumber = {{ fileNumber }};
-	var defaultK 		= {{ defaultK }};
-	var KValue 			= {{ KValue }};
-
-	var colorChartStr   = "{{ colorChartStr }};"
-	var colorChart      = colorChartStr.split("#");
-</script>
-<script>
-	$(document).ready( function () {
-
-	  // If there is data scroll down to the graph on page load
-	  //document.getElementById("graph-anchor").scrollIntoView({block: "start", behavior: "smooth"});
-
-	  // Handle the return to top links
-	  $('.to-top').click(function(){
-	      $("html, body").animate({ scrollTop: 0 }, 800);
-	      return false;
-	  });
-	});
-</script>
-<style type="text/css">
-/*.has-chevron {
-    cursor: pointer;
-}
-.rotate {
-    color: gray;
-    margin-left: 5px;
-    -webkit-transition: all 0.25s ease;
-    -moz-transition:    all 0.25s ease;
-    -o-transition:      all 0.25s ease;
-    transition:         all 0.25s ease;
-}*/
-</style>
-
+    <!-- Lexos K Means Script -->
+    <script type="text/javascript"
+            src="{{ url_for('static', filename='js/scripts_kmeans.js') }}?ver={{ version }}"></script>
+    <!-- Latest plotly js -->
+    <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
+    <!-- Latest compiled and minimized DataTables CSS -->
+    <link rel="stylesheet" type="text/css"
+          href="https://cdn.datatables.net/v/bs/jszip-2.5.0/pdfmake-0.1.32/dt-1.10.16/b-1.5.1/b-colvis-1.5.1/b-html5-1.5.1/b-print-1.5.1/fc-3.2.4/fh-3.1.3/r-2.2.1/datatables.min.css"/>
+    <!-- Latest compiled and minimized DataTables JS -->
+    <script type="text/javascript"
+            src="https://cdn.datatables.net/v/bs/jszip-2.5.0/pdfmake-0.1.32/dt-1.10.16/b-1.5.1/b-colvis-1.5.1/b-html5-1.5.1/b-print-1.5.1/fc-3.2.4/fh-3.1.3/r-2.2.1/datatables.min.js"></script>
+    <!-- Datable plug in to truncate the columns. -->
+    <script type="text/javascript"
+            src="https://cdn.datatables.net/plug-ins/1.10.16/dataRender/ellipsis.js"></script>
```

Comment:
I think you need to include npm version of these packages. See other pages for example

Reply:
Yeah.. I think I may create a separate PR for this.. Since the same line exists in ```similarity.html``` and ```statistics.html```. Same thing for plotly package.